### DISCUSSION TITLE
Welcome to AlphaImpute2 Discussions!

### DISCUSSION THREAD

**Discussion by gregorgorjanc** [AlphaGenes/AlphaImpute2#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Leo4Luffy** [comment]
Hi nice to meet you. My name is Jorge Leonardo. Today I am doing my doctorate in animal genetic improvement. Using some of the software developed by the AlphaGenes team, including AlphaSimR and AlphaImpute2. Very grateful to you!!
